src/rvpt/camera.cpp

```diff
@@ -57,6 +57,9 @@ void Camera::recalculate_values()
     float view_height = tanf(theta / 2);
     float view_width = aspect * view_height;
 
+    translation = glm::vec3(0.784, 1.631, -0.564);
+    rotation = glm::vec3(-82.20, 42, 0);
+
```

Comment:
Is this just to make the camera not start in a weird place?

Reply:
Should be gone now, I peeled the last commit and fixed it.